HHS/Head-Start-TTADP
# TTAHUB-138 Add TouchPoints Survey to AR Landing

### 138

## Description of change

Added survey button to AR Landing screen for TouchPoints survey.

This has changes for both the survey button and CSP directive changes.

## How to test

Go to the AR Landing Screen and click the feedback button.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-138


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

~- [ ] Staging smoke test completed~

### After merge/deploy


### 143

## Description of change
Follow-up PR to the TTA overview widget. Fixes region order in the region select dropdown. Adds non-grantees to the widget and combines duration into one item. Additionally, it includes the legacy reports in statistics and limits the reports to approved ones. This PR also creates a script that allows to extract the non-grantee data from the legacy imported field.


## How to test
- go to sandbox `https://tta-smarthub-dev.app.cloud.gov/activity-reports`
- assign yourself permissions to a lower number region (you might have to remove one first if having permissions to all)
- verify that the available regions in the region selectors are displayed in an ascending order
- add an activity report for a non-grantee
- verify that the widget was updated

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-143


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [n/a] Staging smoke test completed

### After merge/deploy
